Add brief note about using --prerelease flag (#2038)
open-telemetry/opentelemetry.io: content/en/docs/instrumentation/net/getting-started.md

dotnet add package OpenTelemetry.Instrumentation.SqlClient --prerelease
```

> Note that the `--prerelease` flag is required for all instrumentation packages 
> as they are all are pre-release, and using the flag is the only way to get the 
> latest version via the CLI. 

Reviewer: ```suggestion
Note that the `--prerelease` flag is required for all instrumentation packages 
because they are all are pre-release.
```
